iFixit/react-commerce
# Explicitly capture `getServerSideProps` exceptions

- Upgrades our sentry dependency to 7.45.0
- Updates our _error page (copied from Sentry)
- Adds a workaround for https://github.com/getsentry/sentry-javascript/issues/7602 with an explicit try/catch wrapper on `getServerSideProps`

## QA

Assuming #1492 hasn't been closed yet, visit the following url on a Vercel preview

```
/Parts/MacBook_Pro/Display_Components?p=-1%20OR%203%2B947-947-1%3D0%2B0%2B0%2B1
```

and verify that an error shows up in react-commerce-dev similar to [this one](https://ifixit.sentry.io/share/issue/73c0da2fc11f4b2980cbca6420bda4a1/).

Closes #1491
